ob-pattern-1/README.md

```diff
@@ -0,0 +1,302 @@
+# Helm Chart for deployment of WSO2 Open Banking API Manager with Key Manager
+
+Resources for building a Helm chart for deployment of [All-In-One WSO2 Open Banking API Manager with WSO2 Open Banking Key Manager
+and Business Intelligence support](https://docs.wso2.com/display/OB150/Open+Banking+API+Manager+High+Availability+Deployment#OpenBankingAPIManagerHighAvailabilityDeployment-DeploymentPatterns).
+
+![WSO2 Open Banking pattern 2 deployment](https://raw.githubusercontent.com/wso2/kubernetes-open-banking/v1.5.0.1/ob-pattern-1/pattern-1.png)
+
+For advanced details on the deployment pattern, please refer to the official [documentation](https://docs.wso2.com/display/OB150/Configuring+WSO2+Open+Banking).
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Quick Start Guide](#quick-start-guide)
+* [Configuration](#configuration)
+* [Runtime Artifact Persistence and Sharing](#runtime-artifact-persistence-and-sharing)
+* [Managing Java Keystores and Truststores](#managing-java-keystores-and-truststores)
+* [Configuring SSL in Service Exposure](#configuring-ssl-in-service-exposure)
+
+## Prerequisites
+
+* WSO2 product Docker images used for the Kubernetes deployment.
+
+  For a production grade deployment of the desired WSO2 product-version, it is highly recommended to use the relevant
+  Docker image which packages WSO2 Updates, available at [WSO2 Private Docker Registry](https://docker.wso2.com/).
+  
+  In order to use these images, you need an active [WSO2 Subscription](https://wso2.com/subscription).
+  <br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://helm.sh/docs/intro/install/)
+  and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the steps provided in the
+  following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup).<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/).
+
+  Ensure that you have enabled SSL Passthrough support in the Ingress Controller. For instructions on enabling
```

Comment:
Do we need to enable SSL passthrough?

Reply:
@ThilinaManamgoda yes it is required for the Gateway service.